### ISSUE TITLE
testgrid.k8s.io not sending e-mail alerts

### ISSUE THREAD

**Issue by spiffxp** [kubernetes/test-infra#18365]
**What happened**:
I happened to notice https://testgrid.k8s.io/sig-testing-misc#metrics-bigquery has been failing since July 10th, yet I have received no e-mail for it.  It's configured to alert after 2 failures.  Looking around I found other jobs that should have e-mailed alerts (e.g. postsubmits failing in https://testgrid.k8s.io/sig-testing-images)

Looking at https://groups.google.com/forum/#!forum/kubernetes-sig-testing-alerts it seems like alerts stopped being sent out for any jobs configured to send here around mid-June

**What you expected to happen**:
To receive e-mail alerts when the jobs failed

**How to reproduce it (as minimally and precisely as possible)**:
n/a

**Please provide links to example occurrences, if any**:
n/a

**Anything else we need to know?**:

/area testgrid

**Comment by spiffxp** [comment]
/assign @michelle192837 
This is a known issue internally and is being investigated

**Comment by michelle192837** [comment]
Apologies for the delay on this folks, thought a change earlier this week would fix it and it did not.

ALSO, further apologies, alert emails (which I think are correctly identifying failing tabs?) may have been sent by my local testing just now. 😬 I will fix that immediately.

**Comment by michelle192837** [comment]
Ah, false alarm on the sent emails, did seem to be a coincidence. ^^ Looking at the logs for the actual alerter in prod, it seems like we did legitimately find and send alert emails for failing tabs.

I believe this was due to us using an old version of the Summarizer (which alerter relies on), which is now updated thanks to @fejta and @spiffxp from last and this week. Thanks y'all!

(I will follow up at some point this or next week with better alerting so we catch this in future. Apologies again for the trouble!)
